Earthworks-01/readme.md

```diff
@@ -0,0 +1,28 @@
+
+### Intent
+
+This unit test establishes a simple embankment of infrastructure facilities.
+
+### Prerequisites
+
+This scenario builds upon following other scenarios:
+
+- ProjectSetup-1
+- Tin-1
+
+### Content
+
+This scenario covers the following concepts and/or IFC entities:
+
+- IfcEarthworksFill
+- IfcEarthworksFillTypeEnum
```

Comment:
```suggestion
- `IfcEarthworksFill`
- `IfcEarthworksFillTypeEnum`
```

I suggest using the "grave accent" character to enclose IFC entities.

Reply:
Added.